cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -269,10 +335,10 @@ void ImageFlip::imageCallback(const sensor_msgs::ImageConstPtr& color_image_msg)
 				dst -= 6;
 			}
 		}
-		rot_mat.at<double>(0,0) = -1.;
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.